src/utils/plotSpecs/generateFeaturesVsUMIsScatterplot.js

```diff
@@ -1,124 +1,135 @@
-const generateSpec = (config, plotData) => ({
-  $schema: 'https://vega.github.io/schema/vega/v5.json',
-  width: config.dimensions.width,
-  height: config.dimensions.height,
-  autosize: { type: 'fit', resize: true },
-  padding: 5,
+import calculateStdev from '../mathFormulas';
```

Comment:
I would use the same name of the function we are exporting, makes it easier to track: `import stdev from ...`

Reply:
I agree, and most of the libraries just use `std` as name directly pretty standard. Also the file I'd probably just name it `math` which is pretty common too.

Reply:
changed to `stdev`